diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/ASTNodeUtilsTest.kt

```diff
@@ -471,89 +519,80 @@ class ASTNodeUtilsTest {
                 private fun foo(a: Int): Int = 2 * a
             }
         """.trimIndent()
+        var firstCounter = 0
         applyToCode(code){node ->
-            if (node.elementType == MODIFIER_LIST)
+            if (node.elementType == MODIFIER_LIST) {
                 Assert.assertFalse(node.isAccessibleOutside())
+                firstCounter++
+            }
         }
         code = """
             class Test() {
                 /**
                 * test method
                 * @param a - dummy int
                 */
-                fun foo(a: Int): Int = 2 * a
+                public fun foo(a: Int): Int = 2 * a
             }
         """.trimIndent()
+        var secondCounter = 0
         applyToCode(code){node ->
-            if (node.elementType == MODIFIER_LIST)
+            if (node.elementType == MODIFIER_LIST) {
                 Assert.assertTrue(node.isAccessibleOutside())
+                secondCounter++
+            }
         }
+        Assert.assertEquals(1, firstCounter)
+        Assert.assertEquals(1, secondCounter)
     }
     @Test
     fun `test leaveOnlyOneNewLine`() {
         val code = """
             var x = 2
-            
-            
+
+
         """.trimIndent()
+        var counter = 0
         applyToCode(code) { node ->
             if (node.elementType == WHITE_SPACE && node.text.contains("\n\n")) {
                 val parent = node.treeParent
                 val firstText = node.text
                 node.leaveOnlyOneNewLine()
                 var secondText: String? = null
                 parent.getChildren(null).forEach { newNode -> secondText = newNode.text }
```

Comment:
This way you store text from last child node. It gets you correct results, but doesn't represent intentions properly. Instead you could do something like find node with type WHITE_SPACE after node with type PROPERTY.